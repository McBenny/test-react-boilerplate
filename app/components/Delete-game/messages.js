import { messages as HomeMessages } from '../../containers/HomePage/messages';

export const messages = {
    ...HomeMessages,
    title: 'Delete a game',
    question: 'Are you sure you want to delete the following game?',
    caption: 'Match data',
    match: 'Teams and score',
    warning: 'Warning:',
    warningText: 'This cannot be reverted. Any deleted game is definitely erased. Use with caution!',
    cancel: 'Cancel',
    confirm: 'Confirm'
};
