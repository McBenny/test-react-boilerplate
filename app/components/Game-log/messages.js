export const messages = {
    title: 'Game log:',
    lastAction: 'Last action',
    displayFullLog: 'Display full log',
    hideFullLog: 'Hide full log',
    fullLog: 'Full log',
    startGame: 'Start of the game',
    endGame: 'End of the game',
    gamePaused: 'Game paused...',
    gameResumed: '...game resumed',
    timeOutFor: 'Time out requested by',
    startPeriod: 'Start of',
    endPeriod: 'End of',
    period1: '1st half',
    period3: '2nd half',
    period5: '1st half (extra time)',
    period7: '2nd half (extra time)',
    goalFor: 'Goal for',
    penaltyFor: 'Goal (penalty) for',
    yellowCardFor: 'Yellow card for',
    redCardFor: 'Red card for',
    blueCardFor: 'Blue card for',
    suspensionFor: '2 minutes suspension for',
    undo: 'Undo'
};
