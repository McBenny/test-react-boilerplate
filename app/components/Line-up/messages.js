export const messages = {
    title: 'Players',
    maxActionsReached: 'Invalid choice for this player',
    listOfPlayers: 'List of players',
    captainInitial: 'C',
    noPlayers: 'No players, please insert players in "Settings"',
    listOfOfficials: 'List of officials',
    noOfficials: 'No officials, please insert officials in "Settings"'
};
