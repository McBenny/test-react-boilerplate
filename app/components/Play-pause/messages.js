export const messages = {
    title: 'Play-pause screen',
    close: 'Close'
};
