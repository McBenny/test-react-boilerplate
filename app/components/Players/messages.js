/**
 * This entire object is embedded in /app/components/Line-up/messages.js
 * Remove keys with extreme care.
 */
export const messages = {
    title: 'Players',
    titles: {
        'boilerplate/Game/ADD_YELLOW_CARD': 'Add a YELLOW card to:',
        'boilerplate/Game/ADD_SUSPENSION': 'Add a 2 minutes suspension to:',
        'boilerplate/Game/ADD_RED_CARD': 'Add a RED card to:',
        'boilerplate/Game/ADD_BLUE_CARD': 'Add a BLUE card to:'
    },
    cancel: 'Cancel',
    maxActionsReached: 'Invalid choice for this player',
    listOfPlayers: 'List of players',
    none: 'none',
    penalty: 'Is it a penalty?',
    captainInitial: 'C',
    goalieInitial: 'GK',
    noPlayers: 'No players, please insert players in',
    listOfOfficials: 'List of officials',
    noOfficials: 'No officials, please insert officials in',
    settings: 'Settings'
};
