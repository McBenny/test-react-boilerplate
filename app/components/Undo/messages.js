export const messages = {
    title: 'Revert an event',
    introduction: 'Do you want to revert the following event?',
    cancel: 'Cancel',
    yes: 'Yes, revert this event'
};
