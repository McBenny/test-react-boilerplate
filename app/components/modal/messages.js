export const messages = {
    cancel: 'Cancel',
    close: 'Close'
};
