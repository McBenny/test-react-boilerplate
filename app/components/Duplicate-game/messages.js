import { messages as HomeMessages } from '../../containers/HomePage/messages';

export const messages = {
    ...HomeMessages,
    title: 'Duplicate a game',
    question: 'Are you sure you want to duplicate the following game?',
    caption: 'Match data',
    match: 'Teams and score',
    warning: 'Warning:',
    warningText: 'This cannot be reverted. Any deleted game is definitely erased. Use with caution!',
    cancel: 'Cancel',
    confirm: 'Confirm'
};
