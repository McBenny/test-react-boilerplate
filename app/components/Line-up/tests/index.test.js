import React from 'react';
import { render } from '@testing-library/react';

import LineUp from '../index';
import { messages } from '../messages';

const componentProperties = {
    popupVisibility: true,
    team: 'A',
    playersList: [],
    captainId: 0,
    officialsList: [],
    closeHandler: () => {},
    openPopup: () => {}
};

describe('<LineUp />', () => {
    it('should render and match the snapshot', () => {
        const {
            container: { firstChild }
        } = render(<LineUp {...componentProperties} />);
        expect(firstChild).toMatchSnapshot();
    });
});

describe('LineUp messages', () => {
    it('should be defined and have at least a title', () => {
        expect(messages).toBeDefined();
        expect(messages.title).toBeDefined();
        expect(Object.keys(messages).length).toBeGreaterThan(1);
    });
});
