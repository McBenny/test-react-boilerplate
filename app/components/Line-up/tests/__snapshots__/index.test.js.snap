// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LineUp /> should render and match the snapshot 1`] = `
<aside
  class="modal__overlay"
>
  <div>
    <div
      class="modal "
    >
      <div
        class="modal__header"
      >
        <h2
          class="modal__title"
        >
          Line-up
        </h2>
        <button
          type="button"
        >
          Close
        </button>
      </div>
      <div
        class="modal__variable-height"
      >
        <h3>
          List of players
        </h3>
        <p>
          No players, please insert players in
           
          <button
            type="button"
          >
            Settings
          </button>
          .
        </p>
        <h3>
          List of officials
        </h3>
        <p>
          No officials, please insert officials in
           
          <button
            type="button"
          >
            Settings
          </button>
          .
        </p>
        <button
          type="button"
        >
          Cancel
        </button>
      </div>
    </div>
  </div>
</aside>
`;
